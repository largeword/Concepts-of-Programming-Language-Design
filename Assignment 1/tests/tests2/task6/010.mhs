main :: [Int] = 
    let list :: [Int] = recfun list :: [Int] = Cons 3 (Cons 5 (Cons 7 list));
        a :: [Int] = Cons (head (tail (tail (tail (tail (tail (tail list))))))) (Cons (head (tail (tail (tail (tail (tail (tail (tail list)))))))) (Cons (head (tail (tail (tail (tail (tail (tail (tail (tail list))))))))) Nil));
        b :: [Int] = a;
    in Cons (head b) (Cons (head (tail (tail b))) Nil);