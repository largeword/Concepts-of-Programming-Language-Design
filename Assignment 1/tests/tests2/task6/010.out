(Cons 3 (Cons 7 Nil))